### DISCUSSION TITLE
Input Group focus z-index

### DISCUSSION THREAD

**Discussion by rcoopr** [saadeghi/daisyui#957] [Ideas / feature requests]
When an input of an input group is focused, the outline appears in front of all elements prior to it, and behind the elements after it.

This can be observed [here](https://daisyui.com/components/input-group/#label-on-both-sides) Where the focus outline is in front of the price (desired behaviour) and behind the unit (undesirable)

It seems to me that this should be fixed with a stacking context for input groups and putting the input above all others. Perhaps there are technical reasons to avoid this, so I thought I'd start a discussion

**Comment by saadeghi** [comment]
Fixed in version `2.19.1`  
Thank you for reporting.
